assets/javascripts/kitten/components/graphics/icons/cross-icon/index.js

```diff
@@ -2,19 +2,22 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import COLORS from '../../../../constants/colors-config'
 import { computeFromRatio } from '../../../../helpers/utils/ratio'
+import { checkDeprecatedSizes } from '../../../../helpers/utils/deprecated'
 
 const DEFAULT_WIDTH = 8
 const DEFAULT_HEIGHT = 8
 
 export const CrossIcon = ({ color, title, width, height, size, ...props }) => {
+  checkDeprecatedSizes(size)
+  
   const computed = computeFromRatio({
     defaultWidth: DEFAULT_WIDTH,
     defaultHeight: DEFAULT_HEIGHT,
     width,
     height,
   })
 
-  if (size === 'big') {
+  if (size === 'large') {
```

Comment:
Pour éviter les problèmes de déprécation

```suggestion
  if (size === 'large' || size === 'big') {
```